.gitignore

```diff
@@ -3,3 +3,4 @@ node_modules
 build
 dist
 lib
+.idea
```

Comment:
This should be added by people in their own `~/.gitignore`, please remove this from here.